Tests/DecoderTests.swift

```diff
@@ -33,11 +33,11 @@ class ValidDecoder: Decoder {
         self.toDictionary = toDictionary
     }
 
-    func decode(object: AnyObject) throws -> Any {
-        let result = try Parser.parseProperties(json: object) { make in
-            make.propertyForKeyPath("subUInt", type: .uInt)
-            make.propertyForKeyPath("subInt", type: .int)
-            make.propertyForKeyPath("subString", type: .string)
+    func decode(_ object: AnyObject) throws -> Any {
```

Comment:
IMO, the `object` portion of this API didn't provide any additional context and was not necessary. When looking through all the places where this API is actually called, having `object:` doesn't really tell you anything you don't already know.